firebase.json

```diff
@@ -41,7 +41,5 @@
       "enabled": true
     }
   },
-  "extensions": {
-    "firestore-send-email": "firebase/firestore-send-email@0.1.19"
-  }
+  "extensions": {}
```

Comment:
You cannot remove the extension `firestore-send-email` here because it is used in the app